puffin-imgui/src/ui.rs

```diff
@@ -526,7 +528,7 @@ fn paint_record(
                 painter.draw_list.add_text(
                     [
                         start_x + 4.0,
-                        top_y + 0.5 * (options.rect_height - painter.font_size),
+                        0.5f32.mul_add(options.rect_height - painter.font_size, top_y),
```

Comment:
ugh... I am not a big fan of `clippy::suboptimal_flops`. It really hurts readability IMHO :/
Maybe we could just silence the warning for this file with `#![allow(clippy::suboptimal_flops)]` instead and revert this particular line. Or maybe I just need to get used to reading code with `mul_add` :)

Reply:
So, I believe this lint is just an optimization, but if you don't need the optimization then to me readability should win out (I think the other way is more readable). I don't know the code well enough to say, although I do suspect it's not needed. Either way, let me know how you want to go and I'll implement it.

Reply:
I'd like to have `#![allow(clippy::suboptimal_flops)]` at the module level and revert these changes please :)